context.go

```diff
@@ -793,6 +794,14 @@ func (c *Context) parseBlueprintsFile(filename string, scope *parser.Scope, root
 
 	f, err := c.fs.Open(filename)
 	if err != nil {
+		stats, statErr := c.fs.Lstat(filename)
+		if statErr == nil {
+			if stats.Mode()&os.ModeSymlink != 0 {
+				err = fmt.Errorf("%v is a symlink that could not be opened, perhaps its destination doesn't exist: %v", filename, err)
```

Comment:
Can you use readlink to find the destination of the link, stat that, and print a even clearer error if it doesn't exist?

Reply:
Sure. Am I correct to assume that a single ReadLink is all we'd like to do (rather than a recursive ReadLink for a potentially maximally informative error message in the case of a chain of symlinks whose final destination doesn't exist)?

Reply:
Good point, but to handle chained readlinks you would need to deal with loops, which sounds like more trouble than it is worth.